dev/default.env

```diff
@@ -20,3 +20,11 @@ HYMTRUTH_JWT_KEY_PW=
 MASH_JWT_KEY_PW=
 MSTUDY_JWT_KEY_PW=
 RADAR_JWT_KEY_PW=
+
+# generate via: python3 -c 'import os,base64; print(base64.urlsafe_b64encode(os.urandom(32)).decode())'
```

Comment:
Ah, was wondering how you did this. Not sure if it is analogous, but I did instead

```
cat /dev/urandom | tr -dc 'A-Za-z0-9_=' | fold -w 44 | head -n 1
```